apps/event-manager-program/programs/event-manager/src/instructions/event/create_event.rs

```diff
@@ -89,7 +89,8 @@ pub fn handle(
   event_start_date: i64,
   event_end_date: i64,
   ticket_price: u64, 
-  ticket_quantity: u32
+  ticket_quantity: u32,
+  f_id: String
```

Comment:
f_id should be part of the instruction's arguments in the context definition. And should be used in the `seeds` constraint. The main problem is having 2 different IDs.